gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/PolygonDrawingViewModel.java

```diff
@@ -188,7 +186,17 @@ private void addVertex(Point vertex, boolean isNotSelectedByUser) {
     updateDrawnPolygonFeature(ImmutableList.copyOf(vertices));
   }
 
+  @VisibleForTesting
```

Comment:
Can we test using the existing public methods rather than exposing internal state?

Reply:
Fixed